src/tiled_node/octant.cpp

```diff
@@ -221,7 +141,7 @@ void HexMapOctant::bake_mesh() {
 
         Transform3D transform;
         transform.basis = cell->get_basis();
-        transform.set_origin(hex_map.cell_id_to_local(cell_key));
+        transform.set_origin(hex_map.get_cell_center(cell_key));
```

Comment:
When we split get_cell_center() from get_cell_origin(), we'll need to update this.